public/locales/de/translation.json

```
@@ -154,6 +154,39 @@
     "successOffer": {
       "thankYou": "Vielen Dank, wir haben Deine Nachricht weitergeleitet.",
       "toHome": "ZUR STARTSEITE"
+    },
+    "signIn": {
+      "noUser": "Es existiert kein Nutzer mit dieser Email. Bitte registriere dich zuerst.",
+      "wrongUserOrPw": "Falsche Email Adresse oder falsches Passwort angegeben.",
+      "enterEmailForReset": "Bitte fülle das E-Mail Feld aus, um dein Passwort zurück zu setzen.",
+      "pwResetError": "Fehler beim Zurücksetzen des Passworts. Bist du sicher, dass du deine E-Mail korrekt angegeben hast?",
+      "yourEmail": "Deine Emailadresse",
+      "yourPw": "Dein Passwort",
+      "email": "Email",
+      "password": "Passwort",
+      "resetPw": "Passwort zurücksetzen",
+      "loginNow": "Jetzt anmelden",
+      "registerNow": "Neu registrieren",
+      "pwResetConfirmation": "Eine Email mit Anleitung zum Zurücksetzen deines Passworts wurde dir zugesendet!",
+      "headerText": "Melde dich mit deiner E-Mail und deinem Passwort an um {{reasonForSignin}}.",
+      "reasonForSigninDefault": "eine Hilfe-Anfrage zu posten",
+      "emailInvalid": "Bitte gib deine korrekte Emailadresse an."
```

During my tests I watched the `too-many-requests` error and I thought it could be a good idea to catch that as well.

```suggestion
      "emailInvalid": "Bitte gib deine korrekte Emailadresse an.",
      "tooManyRequests": "Das waren zu viele Fehlversuche, bitte versuche es später noch einmal."
```